### PR TITLE
Feat: Add footer

### LINKED ISSUE #68: [Footer] Import the latest footer and update the links
## Description 

This issue covers importing the code for the footer from either the Airgrab ui, or Governance ui codebase.

Additionally, the links will need to be updated, and the darkmode toggle removed

## Related info

- [Design](https://www.figma.com/design/eW63rQE38ql8NGBYcFnCSJ/Analytics-portal?node-id=6857-1177&m=dev)
- [Governance UI Footer](https://github.com/mento-protocol/governance-ui/tree/main/src/components/_shared/footer)

### PR DESCRIPTION
**Pull request by denviljclarke**
### Description

This PR adds the Mento footer component, the same component that is used in all of the other Mento UIs

### Other changes

fixed some header padding

### Tested

Ran the site locally in both desktop and mobile layouts
### Related issues

- Fixes #68

### CODE REVIEW

**Comment by RyRy79261** [comment]
Deployment errors

**Review by bayological — COMMENTED** [review]
All looks good, the discord icon is slightly misaligned:
image

**Review by bayological — COMMENTED** [review]
I also noticed there are some footer links that have been left out(source, team, roadmap, privacy, terms)

Figma:
image

Current:
image

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -1,35 +1,138 @@
...
-        <a className="ml-[10px] p-[10px]" href="/legal/terms">
-          Terms
-        </a>
-        <a className="ml-[10px] p-[10px]" href="/legal/privacy">
-          Privacy
-        </a>
-        <ChangeStory />
+        <MentoLogoIcon />
+        <p className="pt-3 font-inter text-[#636768] ">
+          Mento © 2024. <br />
```
Comment (chapati23) [inline src/components/Footer.tsx:25]: are we changing this from the prior dynamic `year` prop to be hardcoded to 2024 now? is that an intentional change?

**Review by chapati23 — CHANGES_REQUESTED** [review]
- vertical spacing is off for the footer links
- horizontal spacing is off for the social icons
Pasted_Image_26_06_24__15_22

**Review by bayological — APPROVED** [review]

**Pull request merged**